### ISSUE TITLE
Reconsidering Pkg name

### ISSUE THREAD

**Issue by alaindanet** [BecksLab/EcologicalNetworksDynamics.jl#154]
The current package name has been flagged to be may be misleading for two reasons:
1. The package name with a `s` to `Network` in `EcologicalNetworksDynamics` can be considered as a grammar mistake so the `s` can be easily forgotten
2. Change completely package name because it does cover only food-webs currently?

**Comment by iago-lito** [comment]
Not sure what you mean by "flag" here. I don't think anyone did anything to raise official concern regarding the current package name ^ ^"
This being said, this issue is indeed asking to reconsider the package name. Things to bear in mind in this respect:
- The package named was originally discussed and settled in #41.
- The current package name complies with [julia's naming policy](https://pkgdocs.julialang.org/v1/creating-packages/#Package-naming-guidelines).

- Regarding the easily-forgotten middle `s` in `EcologicalNetworksDynamics`:
   - Forgetting it yields a pretty clear error message in julia today.
        ```julia-repl
        pkg> add EcologicalNetworkDynamics
        ERROR: The following package names could not be resolved:
         * EcologicalNetworkDynamics (not found in project, manifest or registry)
           Suggestions: EcologicalNetworksDynamics EcologicalNetworks EcologicalNetworksPlots
        ```
  - The repl does auto-complete the name on `] add Ecol<tab>`, so you can only mistake if you type it manually.
  - There were indeed mistakes in our docs, which could have unfortunately been copy-pasted around, but this has already been fixed today as #153.

- Regarding the package scope:
  - Former name was `BioEnergeticFoodWebs`, because the ODEs simulated were only compliant with this model.
  - More alternate models have been integrated since.
  - Not all models have been integrated since.
  - There are plans to integrate more models: see for instance #134 and #82. This is a work in progress.
  - Still, `EcologicalNetworksDynamics` possibly suggest _more_ than the current capabilities of the package.
- Now that it has been recorded against JuliaRegistries, changing the package name is [neither recommended nor straightforward](https://github.com/JuliaRegistries/General?tab=readme-ov-file#how-do-i-rename-an-existing-registered-package). Actually it cannot exactly be done. We would need to:
   - Deprecate `EcologicalNetworksDynamics.jl`.
   - Pull a new UUID for the package along with its new name.
   - Register as a new package.
   - Migrate this repo to a new address.
   - Update all urls, documentations and users.

My personal opinion:
- If this is mostly about the possibility of mispelling an 's', then I would strongly stand in favour of just teaching people to use copy-paste and autocompletion, be them scientific journal reviewers ;)
- If this is mostly about the package scope, then I would argue that suggesting a wide scope is one good way to commit towards future improvements of the package, which are already being developed. This package name states the ambition.

**Comment by ismael-lajaaiti** [comment]
Can you close this issue? I think we have agreed on keeping the same package name. We can always reopen this, if some referee still complains about the naming.
